docs/source/manifest.rst

```diff
@@ -0,0 +1,72 @@
+==============================================
+Understanding SMaHT Data Portal Manifest Files
+==============================================
+
+
+The preferred method to download data from the SMaHT data portal is to pass
+portal access credentials to the command provided in the ``smaht_manifest`` files.
+
+Most values in the manifest are single valued, but in some select scenarios multi-valued fields are separated by the pipe (``|``) character.
+
+Below are the columns listed in the ``manifest`` files as of the May 2024 data release.
+
+#. **File Download URL** - this URL, when navigated to while passing portal access credentials, will redirect to a pre-signed URL for downloading the associated file provided the user who made the request has access to the file.
+
+#. **File Accession** - this value is a server generated value specific to the SMaHT data portal, generally speaking it is unique with the exception that any associated extra files associated with the root file will have the same accession but different file extension. While you can reference files in data portal by this value, when communicating with others it is best to use the following field **File Name**.
```

Comment:
Do users know what the difference between extra files and root files are? I might add an example saying that given a BAM file ``SMAFI557D2E7.bam``, the corresponding index file is ``SMAFI557D2E7.bam.bai``.